doc/requirements.txt

```diff
@@ -2,8 +2,9 @@ pip
 appdirs
 autograd
 autoray
-jax==0.4.16
-jaxlib==0.4.16
+cachetools
```

Comment:
The reason we need this is that PennyLane isn't installed by default in the sphinx env, so we must explicitly set the dependence here?

Reply:
Yes! Full log can be found: https://github.com/PennyLaneAI/pennylane/actions/runs/13838317452/job/38718984043

Reply:
I may be wrong, but I think the issue here is the lack of a venv and/or a miconfiguration between the pyproject.toml file and setup.py
This dep should be pulled in from the install of PennyLane, and shouldn't need to be specified. I think we can ignore it here, but it may be worth creating an issue.

Reply:
Once we have an issue tracked for this, I'll give a :+1:

Reply:
https://github.com/PennyLaneAI/pennylane/issues/7106
@mlxd Hopefully I put things clear there! Feel free to edit it

Reply:
Perfect, thanks.